Gemfile

```diff
@@ -172,9 +168,7 @@ gem 'actionpack-action_caching' # because we use action caching
 
 gem 'rack-cors'
 
-gem 'ruby2_keywords' # needed because we're backporting code from Rails 6.2
-
-gem 'fx',  git: 'https://github.com/teoljungberg/fx.git', ref: '946cdccbd12333deb8f4566c9852b49c0231a618'
+gem 'fx'
```

Comment:
We were on a specific, old version due to how old of a Rails and Ruby version we were on.